packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -14,7 +14,7 @@ export default {
       receiveShare: false
     },
     reconnection: {
-      enabled: false,
+      enabled: true,
```

Comment:
Let's not enable this yet 😄

Reply:
oops thought we can do in reconnect banch

Reply:
Once we merge the feature branch in, we don't want it enabled.